examples/p2p-webrtc/video-transform/client/ios/SimpleChatbot.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,711 @@
+				90031FA02C616EE700408370 /* Frameworks */,
+				90031FA12C616EE700408370 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = SimpleChatbot;
+			packageProductDependencies = (
+				90383A902D9C357F00D0DDA3 /* PipecatClientIOSSmallWebrtc */,
+			);
+			productName = SimpleChatbot;
+			productReference = 90031FA32C616EE700408370 /* SimpleChatbot.app */;
+			productType = "com.apple.product-type.application";
+		};
+		90031FB22C616EE900408370 /* SimpleChatbotTests */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 90031FCA2C616EE900408370 /* Build configuration list for PBXNativeTarget "SimpleChatbotTests" */;
+			buildPhases = (
+				90031FAF2C616EE900408370 /* Sources */,
+				90031FB02C616EE900408370 /* Frameworks */,
+				90031FB12C616EE900408370 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				90031FB52C616EE900408370 /* PBXTargetDependency */,
+			);
+			name = SimpleChatbotTests;
+			productName = SimpleChatbotTests;
+			productReference = 90031FB32C616EE900408370 /* SimpleChatbotTests.xctest */;
+			productType = "com.apple.product-type.bundle.unit-test";
+		};
+		90031FBC2C616EE900408370 /* SimpleChatbotUITests */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 90031FCD2C616EE900408370 /* Build configuration list for PBXNativeTarget "SimpleChatbotUITests" */;
+			buildPhases = (
+				90031FB92C616EE900408370 /* Sources */,
+				90031FBA2C616EE900408370 /* Frameworks */,
+				90031FBB2C616EE900408370 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				90031FBF2C616EE900408370 /* PBXTargetDependency */,
+			);
+			name = SimpleChatbotUITests;
+			productName = SimpleChatbotUITests;
+			productReference = 90031FBD2C616EE900408370 /* SimpleChatbotUITests.xctest */;
+			productType = "com.apple.product-type.bundle.ui-testing";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		90031F9B2C616EE700408370 /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				BuildIndependentTargetsInParallel = 1;
+				LastSwiftUpdateCheck = 1540;
+				LastUpgradeCheck = 1540;
+				TargetAttributes = {
+					90031FA22C616EE700408370 = {
+						CreatedOnToolsVersion = 15.4;
+					};
+					90031FB22C616EE900408370 = {
+						CreatedOnToolsVersion = 15.4;
+						TestTargetID = 90031FA22C616EE700408370;
+					};
+					90031FBC2C616EE900408370 = {
+						CreatedOnToolsVersion = 15.4;
+						TestTargetID = 90031FA22C616EE700408370;
+					};
+				};
+			};
+			buildConfigurationList = 90031F9E2C616EE700408370 /* Build configuration list for PBXProject "SimpleChatbot" */;
+			compatibilityVersion = "Xcode 14.0";
+			developmentRegion = en;
+			hasScannedForEncodings = 0;
+			knownRegions = (
+				en,
+				Base,
+			);
+			mainGroup = 90031F9A2C616EE700408370;
+			packageReferences = (
+				90383A8F2D9C357F00D0DDA3 /* XCLocalSwiftPackageReference "../../../../../../ios-public-repos/pipecat-client-ios-small-webrtc" */,
```

Comment:
We need to update this to use the one from this PR once it's published:
- https://github.com/pipecat-ai/pipecat-client-ios-small-webrtc/pull/1

For now, I'm using my local version.